src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -68,7 +67,10 @@ class ErgoWalletActor(settings: ErgoSettings,
   // The state context is being updated by listening to state updates.
   private def stateContext: ErgoStateContext = storage.readStateContext
 
-  private var height: Int = ErgoHistory.GenesisHeight
+  /**
+    * Height of full-blocks chain as reported by the state processor
```

Comment:
Please define `full-blocks chain` or give reference to the definition.
Also, what is `state processor`, if it is a class use '[[ClassReference]]` syntax of ScalaDoc. If not, then please define.

Reply:
No need for the definition of basic blockchain glossary here I suppose. Headers-chain and full-blocks chain are everywhere around the space, Ergo API and Ergo code. The historically first reference would be the Bitcoin whitepaper (SPV section of it), but I can hardly imagine anyone reading this code before the whitepaper.

Reply:
On the state processor, changed the comment.

Reply:
Well, I give my feedback about code readability in places it is not clear for me. I think it will also be not clear for others.

Again, you better put your explanation above into the code directly. 
(Don't assume people think exactly the way you think, so better to me more explicit in the code)

Reply:
It is better to avoid subjective views when reviewing, as the goal is to improve the code towards some inter-subjective goals. Then you need to clearly understand which group's inter-subjective views you do represent.
I suppose that the comments in the wallet are made not for a person doing a review by skimming through GitHub changes, rather for a person willing to investigate how the wallet works (maybe, in order to make changes then). Supposedly such a person already knows what basic things about Ergo architecture, node settings in regards with different regimes, maybe Scorex tutorial and Yellow paper draft.

Thus providing comments about sync or basic cryprocurrency jargon are just distracting from the goal. So I still do not see a reason for that.